core/templates/pages/oppia-root/app-error-handler.ts

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Error handler for frontend.
+ */
+
+// eslint-disable-next-line oppia/disallow-httpclient
```

Comment:
Any time you use eslint-disable, could you please add an explanatory comment so that other people know that this is not a pattern to follow?

Reply:
Done